<?xml version="1.0" encoding="UTF-8"?><testrun name="MybatisStudentRepositoryTest" project="lyceum-website" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="com.lyceum.dao.mybatis.MybatisConnectionManagerTest" time="0.0">
    <testcase name="testAddStudent" classname="com.lyceum.dao.mybatis.MybatisConnectionManagerTest" time="0.0">
      <failure>junit.framework.AssertionFailedError: Method "testAddStudent" not found&#13;
	at junit.framework.Assert.fail(Assert.java:50)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:161)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
  </testsuite>
</testrun>
